package.json

```diff
@@ -18,42 +18,20 @@
     "build:production": "ember build --environment=production",
     "prepublish": "bower install && npm run-script build:production",
     "start": "ember serve",
-    "test": "testem -R dot ci",
-    "publish-build": "npm run build:production && ./bin/publish_to_s3.js",
-    "test:local": "testem -R dot ci",
-    "test:beta": "testem -f config/testem-beta.json -R dot ci",
-    "test:canary": "testem -f config/testem-canary.json -R dot ci",
-    "test:stable": "testem -f config/testem-stable.json -R dot ci"
+    "pretest": "ember build",
+    "test": "bin/run-tests.js",
+    "publish-build": "npm run build:production && ./bin/publish_to_s3.js"
   },
   "devDependencies": {
     "aws-sdk": "~2.0.0-rc8",
     "bower": "~1.3",
-    "broccoli-compile-modules": "^1.1.0",
-    "broccoli-concat": "0.0.12",
-    "broccoli-defeatureify": "^0.3.0",
-    "broccoli-env": "0.0.1",
-    "broccoli-es3-safe-recast": "0.0.8",
-    "broccoli-es6-module-transpiler": "^0.5.0",
-    "broccoli-es6-transpiler": "^0.1.0",
-    "broccoli-file-creator": "^0.1.0",
-    "broccoli-file-mover": "~0.2.0",
-    "broccoli-jshint": "^0.5.1",
-    "broccoli-merge-trees": "^0.1.4",
-    "broccoli-render-template": "0.0.3",
-    "broccoli-static-compiler": "^0.2.1",
-    "broccoli-replace": "~0.2.0",
-    "broccoli-uglify-js": "^0.1.3",
-    "broccoli-wrap": "0.0.2",
-    "broccoli-yuidoc": "^1.3.0",
+    "chalk": "~0.4.0",
     "defeatureify": "~0.1.4",
     "ejs": "^1.0.0",
-    "ember-cli": "^0.1.4",
+    "ember-cli": "~0.1.11",
     "ember-publisher": "0.0.6",
-    "es6-module-transpiler": "^0.9.5",
-    "es6-module-transpiler-amd-formatter": "^0.2.4",
-    "es6-module-transpiler-package-resolver": "^1.0.1",
+    "emberjs-build": "file:../emberjs-build",
```

Comment:
@rwjblue this needs to be replaced, obviously, but this PR depends on https://github.com/emberjs/emberjs-build/pull/65